packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -543,9 +541,9 @@ mixin AutoCompleteSearchControllerMixin on SearchControllerMixin {
     TextSelection selection, {
     bool handleFields = false,
   }) {
-    String activeWord;
-    String leftSide;
-    String rightSide;
+    String? activeWord;
+    String? leftSide;
+    String? rightSide;
```

Comment:
these can all be late and non-nullable

Reply:
they cannot be late, because they are initialized after 'if'. Assigned default values to them.